packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
  */
 
+import crypto from 'crypto';
```

Comment:
Let's use `crypto-js` for node/browser compatibility.

```suggestion
import crypto from 'crypto-js';
```